packages/react-fs/src/ReactFilesystem.js

```diff
@@ -91,6 +92,121 @@ function checkPathInDev(path: string) {
   }
 }
 
+function createAccessCache(): Map<string, Array<number | Record<void>>> {
+  return new Map();
+}
+
+export function access(path: string, mode?: number): void {
```

Comment:
I honestly don't know why this would be useful in a webapp. But it was there so I added it.